jax/_src/lax/linalg.py

```diff
@@ -131,7 +134,8 @@ def cholesky(x, *, symmetrize_input: bool = True):
   return jnp.tril(cholesky_p.bind(x))
 
 @_warn_on_positional_kwargs
-def eig(x, *, compute_left_eigenvectors=True, compute_right_eigenvectors=True):
+def eig(x: ArrayLike, *, compute_left_eigenvectors: bool = True,
+        compute_right_eigenvectors: bool = True) -> Tuple[Array, Array]:
```

Comment:
Does this not return a tuple of 3 arrays (also dependent on the values of the `compute_left/right` kwargs)?

Reply:
Good catch - fixed!